packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -446,10 +446,15 @@ export class DumbTriggerManager extends Component {
         uri: get(konnector, 'vendor_link')
       })
 
-      if (ciphers.length === 0) {
-        this.showAccountForm()
-      } else {
-        this.showCiphersList(ciphers)
+      // When we fetched the ciphers while the user is already on the account
+      // form step, we don't want to automatically redirect him on the ciphers
+      // list
+      if (this.state.step !== 'accountForm') {
```

Comment:
Instead of doing a negative boolean check, which will fail as soon as a step is added, could we reverse the condition to be positive ? Which step are we assuming here ?

Reply:
See https://github.com/cozy/cozy-libs/commit/ba0469679de1962bd9d2505d53479d0a985ae81d